training/DeepSpeed-Domino/domino/language_model.py

```diff
@@ -127,6 +127,7 @@ def __init__(self,
         self.init_method = config.init_method
         self.encoder_attn_mask_type = encoder_attn_mask_type
         self.encoder_hidden_state = None
+        self.input_split_dim = config.input_split_dim
```

Comment:
one possible thing we need to verify and check is if split dim is seq, can we still initialize the same rope embedding, which is a very popular position embedding function. because for rope:
```
        if self.use_rotary_position_embeddings:
            self.seq_length = args.seq_length
 ```           
  our self.seq_length may not be correct ( ours might be half of original seq_length).